content/en/security/cspm/detection_rules.md

```diff
@@ -0,0 +1,89 @@
+- [**Infrastructure configuration**][5]: These detection rules analyze your containers and Kubernetes clusters to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes. For example, the [/etc/default/docker file permissions are set to 644 or more restrictively][6] rule evaluates Docker file permissions running on a host.
+
+## Customize how your environment is scanned by each rule
+
+Customization of a cloud configuration query directly is not supported at this time, but you can customize how your environment is scanned by each rule.
+
+On the [Rules][13] page, select a rule to open its details page. Under **Exclude benign activity with suppression queries**, set the filtering logic for how the rule scans your environment.
+
+For example, you can exclude resources tagged with `env:staging` using the **This rule will not generate a finding if there is a match with any of the following suppression queries** function. You can also limit the scope for a certain rule to resources tagged with `compliance:pci` using the **Only generate a finding if there is a match with any of the following queries** function.
+
+After you customize a rule, click **Update Rule** at the bottom of the page to apply your changes.
+
+{{< img src="security/cspm/frameworks_and_benchmarks/never-trigger-a-finding-2.png" alt="Customize how your environment is scanned by selecting tags to include or exclude from a rule's scope" >}}
+
+## Set notification targets for detection rules
+
+You can send real-time notifications when a new misconfiguration is detected in your environment by adding notification targets. The complete list of notification options are:
+
+- [Slack][14]
+- [Jira][15]
+- [PagerDuty][16]
+- [ServiceNow][17]
+- [Microsoft Teams][18]
+- [Webhooks][19]
+- Email
+
+On the [Rules][13] page, select a rule to open its details page. In the **Set severity and notifications** section, configure zero or more notification targets for each rule case. You cannot edit the preset severity. See [Notifications][7] for detailed instructions on configuring notifications for detection rules.
+
+Alternatively, create [notification rules][21] that span across multiple detection rules based on parameters such as severities, rule types, rule tags, signal attributes, and signal tags. This allows you to avoid having to manually edit notification preferences for individual detection rules.
+
+{{< img src="security/cspm/frameworks_and_benchmarks/notification-2.png" alt="The Set severity and notifications section of the rule details page" >}}
+
+## Create custom rules
+
+To extend the rules being applied to your environment to evaluate your security posture, you can create custom rules from scratch. You can also clone the default detection rules and edit the copies (GCP only). See [Custom Rules][20] for more information.
```

Comment:
I was trying to find a better way to phrase this first sentence without the idiom "from scratch" but then realized it's kind of a jargon-y sentence. This isn't the best edit of it, maybe you have a better idea how to make this sentence more understandable. 
```suggestion
You can create custom rules to extend the rules being applied to your environment to evaluate your security posture. You can also clone the default detection rules and edit the copies (GCP only). See [Custom Rules][20] for more information.
```